Cargo.toml

```diff
@@ -3,7 +3,7 @@ authors = ["Anthony DiMarco <ocramida@gmail.com>", "Jimmy Cuadra <jimmy@jimmycua
 build = "build.rs"
 description = "AWS SDK for Rust"
 documentation = "http://rusoto.github.io/rusoto/rusoto/index.html"
-exclude = [".gitignore", ".travis.yml", "docgen.sh"]
+exclude = [".gitignore", ".travis.yml", "docgen.sh", "helpers/**/*"]
```

Comment:
Does this mean the codegen subcrate was being included in our previous releases?  Should that be added to the `exclude` section?